downloads/swan-lake-archived.md

```diff
@@ -19,10 +19,22 @@ permalink: /downloads/swan-lake-archived/
         <div class="col-xs-12 col-sm-16 col-md-12 col-lg-12">
             <div class="cStandaloneInstallers" id="archived-versions">
                 <h2>Swan Lake Archived Versions</h2>
+                <div class="cInstallers" style="height:150px"> 
+                <h3 class="release-version">swan-lake-preview6</h3>
+<div class="releases-row">
+    <div class="col-xs-12 col-sm-16 col-md-6 col-lg-6 cLeftTable">
+
+This release is an internal release only.
```

Comment:
@praneesha, should this be "This release is an internal-only release." or "This release is internal only." instead?

Reply:
This is done via https://github.com/ballerina-platform/ballerina-dev-website/pull/1565/files.